### PR TITLE
Feat/fee 429 bar chart tooltip

### PR DESCRIPTION
**Pull request by apryka**
https://metakeen.atlassian.net/browse/FEE-429

### CODE REVIEW

File: packages/charts/src/charts/bar/bar-tooltip-content.component.tsx [hunk 1]
```diff
@@ -44,46 +48,68 @@ const BarTooltip: FC<Props> = ({
...
-      {isList ? (
-        <BulletList
-          items={selectors.map(({ color, selector }) => ({
-            data: getLabel({
+  const [firstSelector] = selectors;
+  const [index] = firstSelector.selector;
+
+  const scaleLabel =
+    xScaleSettings.precision && typeof index === 'number'
+      ? formatScaleLabel(data[index][labelSelector], xScaleSettings)
```
Comment (maciejrybaniec) [inline packages/charts/src/charts/bar/bar-tooltip-content.component.tsx:66]: `format` label based on on scale settings utility provided from **dataviz** layer

File: packages/charts/src/charts/bar/bar-chart.component.tsx [hunk 2]
```diff
@@ -205,6 +210,7 @@ export const BarChart: FC<Props> = ({
                   selectors={tooltip.selectors}
                   isList={tooltip.selectors.length > 1}
                   formatValue={formatTooltip}
```
Comment (maciejrybaniec) [inline packages/charts/src/charts/bar/bar-chart.component.tsx:212]: Use `TooltipFormatter` from `@keen.io/charts-utils` and format value with `formatValue()`.

File: packages/charts/src/components/tooltip-content/tooltip-content.component.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  /** Label for scale */
+  scaleLabel?: string | number;
+  /** Total value */
+  totalValue?: string;
+  /** Percent value */
+  percentValue?: number;
+};
+
+const TooltipContent: FC<Props> = ({
+  isList,
```
Comment (maciejrybaniec) [inline packages/charts/src/components/tooltip-content/tooltip-content.component.tsx:31]: `isList is not required`

File: packages/charts/src/components/tooltip-content/tooltip-content.component.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  items,
+  scaleLabel,
```
Comment (maciejrybaniec) [inline packages/charts/src/components/tooltip-content/tooltip-content.component.tsx:33]: Rename to not couple it with charts that uses `scale`

**Review by maciejrybaniec — CHANGES_REQUESTED** [review]

**Review by maciejrybaniec — APPROVED** [review]

File: packages/charts/src/charts/bar/bar-tooltip-content.component.tsx [hunk 5]
```diff
@@ -44,46 +46,73 @@ const BarTooltip: FC<Props> = ({
...
+  labelSelector,
+  maxWidth,
   formatValue,
 }) => {
-  const {
-    theme: { tooltip },
-  } = useContext(ChartContext) as ChartContextType;
+  const { xScaleSettings } = useContext(ChartContext) as ChartContextType;
+  const { precision, formatLabel } = xScaleSettings;
+  console.log({ precision, formatLabel });
```
Comment (mczapkowicz) [inline packages/charts/src/charts/bar/bar-tooltip-content.component.tsx:55]: You missed console.log() ;)

**Review by maciejrybaniec — APPROVED** [review]

**Pull request merged**